hack/install-buildx

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+topdir="$(realpath $(dirname "$0")/..)"
+cd "${topdir}"
+
+VERSION="v0.4.2"
+BINDIR="$(pwd)/bin"
+DEST="${BINDIR}/buildx"
+
+os=""
+case "$(uname)" in
+  Linux)
+    os="linux"
+    ;;
+  Darwin)
+    os="darwin"
+    ;;
+  *)
```

Comment:
We do have windows releases as well. Not sure how well these scripts would work on windows though.

Reply:
Is there any environment that returns `Windows` as `uname`?

Reply:
Do we want to support MSYS and Cygwin?

Reply:
(I guess it can be a separate PR anyway)